### DISCUSSION TITLE
[Suggestion] Ability to ping multiple selected nodes

### DISCUSSION THREAD

**Discussion by jeffkjolley** [zwave-js/zwave-js-ui#2004] [Ideas]
First- A big thank you to all of the developers. This project has finally made Z-Wave reliable and usable for me.

Just an idea: More than once I've wished for the ability to ping multiple nodes from the Control Panel list view. Ideally by selecting a few or several nodes from the list and clicking a "Ping Selected Nodes" option. 

My network has some mild signal issues (a few switches shielded in metal boxes that I really can't do much about), and I've found that if I ping the nodes prior to doing a network heal, there is a higher chance of successfully healing all of my nodes.

Again, thank you.

**Comment by robertsLando** [comment]
@jeffkjolley You could easilly do that using a custom driver function :)
